Cargo.toml

```diff
@@ -127,7 +127,7 @@ http = "0.2"
 typetag = "0.1"
 toml = "0.4"
 syslog = "5"
-syslog_loose = { version = "0.4.0", optional = true }
+syslog_loose = { version = "0.4.0" }
```

Comment:
Could this stay optional and as a dependency of `transforms-remap`?

Reply:
Unfortunately not. Remap (the mapping module) is not under the `transform-remap` feature flag, or indeed any flag. The idea is ultimately to use remap in more places than just in the transform. It does mean that anything the remap language pulls in will always be there.. We do need to be careful about pulling too much in as more things move to the language. 

Do you think it would be worth creating a feature flag for Remap?

Reply:
Aha, I see. I remembered a small discussion in discord around that (https://discordapp.com/channels/742820443487993987/746070604192415834/759015177264955403). I do think it'd be nice to have remap functionality be, itself, an optional feature -- especially as it grows dependencies like this one -- to help keep compile times down when targeting individual features.

It would then be, also, be an optional feature of components.

Reply:
We can open a follow-up issue for that though, if we want to pursue it.

Reply:
Raised here: #4659